app.js

```diff
@@ -22,4 +24,29 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then((beersArr) => {
+      const data = {
+        listOfBeers: beersArr
+      }
+      console.log(data);
+        res.render('beers', data);
+      })
+      .catch((err) => console.log("Error fetching one beer", err))
+    });
+
+    app.get('/random-beer', (req, res) => {
+      punkAPI.getRandom()
+        .then((responseFromAPI) => {
+          const data = {
+            randomBeer: responseFromAPI
```

Comment:
Here I would pass the element and not the whole array, that way you could skip the `#each` on the random-beer.hbs

```suggestion
            randomBeer: responseFromAPI[0]
```